diagrams/media.get.sequenceDiagram.plantuml

```diff
@@ -0,0 +1,15 @@
+@startuml get media
+hide footbox
+
+
+actor enduser
+participant mediaService
+participant provider
+
+
+enduser -> mediaService: masked URL
```

Comment:
Shouldn't this be uuid instead of masked URL as we discussed today?

Reply:
/api/media/fileId = masked url
I meant to write it masked url as it is sent from the browser automatically without changing the fileId or (uuid) from the front-end side